db/migrations/20180912171047_create_cat_file.sql

```diff
@@ -34,6 +35,8 @@ CREATE INDEX cat_file_flip_log_index
     ON maker.cat_file_flip (log_id);
 CREATE INDEX cat_file_flip_ilk_index
     ON maker.cat_file_flip (ilk_id);
+CREATE INDEX cat_file_flip_msg_sender_index
+    ON maker.cat_file_flip (msg_sender);
```

Comment:
I wonder if we want to either include a `DROP INDEX maker.cat_file_chop_msg_sender_index;` or remove the `DROP INDEX` statements from this migration for consistency.

Reply:
Looks like somebody (Rob) took care of this on a rebase.